src/main/java/frc/trigon/robot/subsystems/sideshooter/SideShooterCommands.java

```diff
@@ -0,0 +1,23 @@
+package frc.trigon.robot.subsystems.sideshooter;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+
+public class SideShooterCommands {
+
+    public static SideShooterCommands getInstance() {
+        return getInstance();
+    }
+
+
+    public Command getSetTargetShooterAngleCommand(Rotation2d targetAngle) {
+        return new FunctionalCommand(
+                () -> SideShooter.getInstance().generateAngleMotorProfile(targetAngle),
+                () -> getSetTargetShooterAngleCommand(targetAngle),
+                (interrupted) -> {},
+                () -> false,
+        SideShooter.getInstance()
+        );
+    }
```

Comment:
Make this static, and instead of typing `SideShooter.getInstance()` every time have a class variable for it. 
Also, add the commands to set the target voltage and target state.